code/game/objects/structures/hybrisa_props.dm

```diff
@@ -1911,10 +1950,46 @@
 /obj/structure/prop/hybrisa/misc/firehydrant
 	name = "fire hydrant"
 	desc = "A fire hydrant public water outlet, designed for quick access to water."
+	icon = 'icons/obj/structures/props/hybrisarandomprops.dmi'
 	icon_state = "firehydrant"
 	density = TRUE
 	anchored = TRUE
-/obj/structure/prop/hybrisa/misc/bench
+	health = 250
+
+/obj/structure/prop/hybrisa/misc/firehydrant/bullet_act(obj/projectile/P)
+	var/damage = P.damage
+	health -= damage
```

Comment:
You could just do `health -= P.damage`

Reply:
changed for all instances in the file now.